src/qcodes/instrument_drivers/tektronix/AWG70000A.py

```diff
@@ -388,6 +389,13 @@ def setSequenceTrack(self, seqname: str, tracknr: int) -> None:
                                    f'CASSet:SEQuence "{seqname}"'
                                    f', {tracknr}')
 
+    def clearAsset(self) -> None:
```

Comment:
```suggestion
    def clear_asset(self) -> None:
```

Functions and methods should be camelcase in new code